.circleci/config.yml

```diff
@@ -1270,18 +1271,23 @@ jobs:
                 text: "Release branch has been cut! :git-fork:"
       - when:
           condition:
-            equal: [ release-to-service-providers, << parameters.target >> ]
+            equal: [ execute-proposal, << parameters.target >> ]
           steps:
             - slack-success:
-                text: "Deployment Released to Service Providers! :gift:"
-                detail: "New Governance Proposals are out. :vote:"
-                slack_mentions_user: "@Dheeraj @ray @roneil @Forrest"
+                text: "Governance Proposal has been executed! :done:"
       - when:
           condition:
-            equal: [ execute-proposal, << parameters.target >> ]
+            equal: [ release-to-service-providers, << parameters.target >> ]
           steps:
             - slack-success:
-                text: "Governance Proposal has been executed! :done:"
+                text: "Deployment Released to Service Providers! :gift:"
+                detail: "New Governance Proposals are out. :vote:"
+                slack_mentions_user: "@Dheeraj @ray @roneil @Forrest"
```

Comment:
perhaps we can set these via CI env vars too

Reply:
Sound good!

Updated: CircleCI.Contexts.slack-secrets.SLACK_RELEASE_MENTIONS